recipe/meta.yaml

```diff
@@ -39,7 +42,7 @@ test:
     - agate.table
     - agate.tableset
   commands:
-    - pip check  # [unix]
+    - pip check
```

Comment:
i'm mostly curious as to what is missing on windows. maybe things have changed since the recipe was first written.

Reply:
pip does not find `tzdata` even if it is installed explicitly: 
```
(%PREFIX%) %SRC_DIR%>pip check 
agate 1.9.1 requires tzdata, which is not installed.
```

Reply:
conda-forge tzdata seems to be something else:
https://github.com/conda-forge/tzdata-feedstock/blob/main/recipe/meta.yaml